Bottom: 7ddcef49ba6a7c357b758b02cc09f9d5a8f15ee5
Top:    324dca7a9df2b8ab8a6b959820c1e86638cb8b4e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:06:45 +0000

Refresh of revert-debug-stuff-looking-into-bad

---

diff --git a/src/polygon1.c b/src/polygon1.c
index cc60364..9089395 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -161,7 +161,7 @@ pline_dump (VNODE * v)
   while ((v = NEXT_VERTEX(v)) != s);
 }
 
-/* static */ void
+/*static */void
 poly_dump (POLYAREA * p)
 {
   POLYAREA *f = p;
